docs/content/Checkbox.mdx

```diff
@@ -85,22 +86,59 @@ An `indeterminate` checkbox state should be used if the input value is neither t
 </>
 ```
 
-## Component props
+## Props
 
-Native `<input>` attributes are forwarded to the underlying React `input` component and are not listed below.
+<PropsTable>
```

Comment:
The heading of prop tables is usually the name of the component: 

```suggestion
### Checkbox

<PropsTable>
```

Reply:
The example heading in the Issue description [here](https://github.com/primer/react/issues/1701) shows it as Props, which feels sensible given the content it describes... or am I misunderstanding this? What was the reasoning behind repeating the component name here?